test/lit/passes/O1_skip_passes.wast

```diff
@@ -0,0 +1,71 @@
+;; RUN: foreach %s %t wasm-opt -O2 --skip-pass=coalesce-locals --skip-pass=simplify-locals --skip-pass=simplify-locals-nostructure -S -o - 2>&1 | filecheck %s
```

Comment:
Please use the comment that makes the auto-updater script handle the file, see e.g.

```
https://github.com/WebAssembly/binaryen/blob/05631b1f108ca1964fa92867205f796f167015e9/test/lit/passes/stack-ir-dce.wast#L1
```
Then running `./scripts/update_lit_checks.py test/lit/passes/O1_skip_passes.wast` will update the file automatically.